packages/dx-react-grid-material-ui/src/plugins/table-header-row.jsx

```diff
@@ -27,7 +33,10 @@ export class TableHeaderRow extends React.PureComponent {
 }
 
 TableHeaderRow.Cell = TableHeaderCell;
+TableHeaderRow.Content = Content;
```

Comment:
As for me, this name isn't associated with a header **cell** content.

Reply:
We use such a name in the TableDetailRow plugin

Reply:
Detail row has only one cell and doesn't look like a common table row. It's just a place for a content.   I can't imagine what `TableHeaderRow.Content` means. Is it a content for `tr`? Looks very strange for me. How can I know that is a content for `th`?